### DISCUSSION TITLE
Accident in the three-node scheme

### DISCUSSION THREAD

**Discussion by stallion2303** [seaweedfs/seaweedfs#2234] [General]
A three-node system has been created. 
`weed server -filer.peers=ip1:8888,ip2:8888,ip3:8888 -master.peers=ip1:9333,ip2:9333,ip3:9333`
An emergency occurs and only one node remains. How do I put a live node in read-only mode? Now the output is: raft.Server: Not current leader

**Comment by chrislusf** [comment]
use `weed server` on the remaining server.

**Reply by stallion2303** [reply]
Do I understand you correctly to remove the *.peers arguments and restart the weed server?

**Reply by chrislusf** [reply]
right.

**Reply by stallion2303** [reply]
And how to start the `weed server` in read-only mode?
